client/src/features/form/FormCreateEvent.js

```diff
@@ -1,14 +1,23 @@
 import { useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
Minor nitpick. It would be better like this:

```suggestion
import { useContext, useState, useEffect } from "react";
```